### PR TITLE
chore: configure 2 dev-preprod deployments, one compatible with Lace …

### PR DESCRIPTION
**Pull request by gytis-ivaskevicius**
# Context 
This is how mainnet deployment is going to be configured, ill add question comments in PR to make sure. We can deploy this from branch or once its merged

### CODE REVIEW

File: nix/cardano-services/deployments/default.nix [hunk 1]
```diff
@@ -308,43 +308,62 @@ in
       "dev-preprod@us-east-1@v1" = final: {
```
Comment (gytis-ivaskevicius) [inline nix/cardano-services/deployments/default.nix:308]: This represents our existing live-mainnet deployment. configuration is identical aside cluster/network changes

File: nix/cardano-services/deployments/default.nix [hunk 2]
```diff
@@ -308,43 +308,62 @@ in
...
 
         values = {
           network = "preprod";
           region = "us-east-1";
 
-          blockfrost-worker.enabled = true;
-          pg-boss-worker.enabled = true;
-
+          backend.hostnames = ["backend.${final.namespace}.eks.${baseUrl}" "${final.namespace}.${baseUrl}"];
+          backend.passHandleDBArgs = false;
```
Comment (gytis-ivaskevicius) [inline nix/cardano-services/deployments/default.nix:363]: Should we pass these arguments? If so this requires handle projector https://github.com/input-output-hk/cardano-js-sdk/blob/master/nix/cardano-services/deployments/backend.provider.nix#L60-L79

File: nix/cardano-services/deployments/default.nix [hunk 3]
```diff
@@ -308,43 +308,62 @@ in
...
+          # blockfrost-worker.enabled = true;
```
Comment (gytis-ivaskevicius) [inline nix/cardano-services/deployments/default.nix:364]: Should we use blockfrost worker from old or new deployment?

**Review by gytis-ivaskevicius — COMMENTED** [review]
Do we require any extra services for new deployment?

**Review by pczeglik-iohk — APPROVED** [review]

**Review by bernokl — DISMISSED** [review]

**Review by pczeglik-iohk — APPROVED** [review]

**Comment by pczeglik-iohk** [comment]
@gytis-ivaskevicius Tests executed. We can merge it

**Pull request merged**